aws/alarms/athena.tf

```diff
@@ -94,6 +94,98 @@ resource "aws_athena_data_catalog" "dynamodb" {
   type        = "LAMBDA"
 
   parameters = {
-    "function" = data.aws_lambda_function.existing.arn
+    "function"   = data.aws_lambda_function.existing.arn
+    "LambdaRole" = aws_iam_role.athena_dynamodb_role.arn
```

Comment:
Just out of curiosity, where did you find this `LambdaRole` parameter since I'm not seeing it in the DataCalalog docs:
https://docs.aws.amazon.com/athena/latest/APIReference/API_DataCatalog.html

Reply:
good catch, this should be associated to the connector app